Common/src/gauss_structure.cpp

```diff
@@ -75,6 +75,23 @@ CGaussVariable::~CGaussVariable(void) {
 
 }
 
+CProperty::CProperty(void) {
+
+  iMat_Prop = 0;
+
+}
+
+CProperty::CProperty(unsigned long valMat_Prop) {
+
+  iMat_Prop = valMat_Prop;
```

Comment:
You can use initializer lists for this kind of stuff, it's regarded as the preferred way of doing it.